### ISSUE TITLE
RPC distortion and camera center

### ISSUE THREAD

**Issue by oleg-alexandrov** [NeoGeographyToolkit/StereoPipeline#247]
The RPC lens distortion model assumes the origin to be at pixel (0, 0) rather than at the image optical center. This does not make it less expressive, but makes it trickier to interpret the result. It may be helpful to make it more compatible with our other distortion models which use the optical center. For now, the convert_pinhole_model tool should still convert any camera model to RPC distortion with no issue though.

**Comment by oleg-alexandrov** [comment]
You can read some of the code to see how Brown Conrady distortion is
handled vs RPC distortion, by comparing, for example:

https://github.com/visionworkbench/visionworkbench/blob/master/src/vw/Camera/LensDistortion.cc#L388

vs

https://github.com/visionworkbench/visionworkbench/blob/master/src/vw/Camera/LensDistortion.cc#L696

Note that for Brown-Conrady the undistortion is given by a formula, so the
distortion has to invert that nonlinear function, while for RPC we fit RPC
polynomials for both distortion and distortion.

The AdjustableTsaiLensDistortion, also in there, is how things are done
more traditionally in the computer vision community. There exist radial and
tangential distortion, and it is the distortion operation which is given by
a formula, and then undistortion has to invert that.

**Comment by oleg-alexandrov** [comment]
Thanks, good points. I modified the Tsai distortion formula to add the
shift which was missing at the end.

There's more to be done, in fact some very careful work needs to be done to
cross-reference these formulas with the code and literature references,
which I don't have time for now. I added a blurb however referring the
reader to the right code, should somebody wish to get to the bottom of
things.
